File: app/src/main/java/closer/vlllage/com/closer/store/models/Phone.java
package closer.vlllage.com.closer.store.models;

import io.objectbox.annotation.Entity;

@Entity
public class Phone extends BaseObject {
    private String name;
    private String status;
    private String photo;
    private Double latitude;
    private Double longitude;

    public String getName() {
        return name;
    }

    public Phone setName(String name) {
        this.name = name;
        return this;
    }

    public String getStatus() {
        return status;
    }

    public Phone setStatus(String status) {
        this.status = status;
        return this;
    }

    public Double getLatitude() {
        return latitude;
    }

    public Phone setLatitude(Double latitude) {
        this.latitude = latitude;
        return this;
    }

    public Double getLongitude() {
        return longitude;
    }

    public Phone setLongitude(Double longitude) {
        this.longitude = longitude;
        return this;
    }

    public String getPhoto() {
        return photo;
    }

    public Phone setPhoto(String photo) {
        this.photo = photo;
        return this;
    }
}

File: app/src/main/java/closer/vlllage/com/closer/store/models/Phone.kt
package closer.vlllage.com.closer.store.models

import closer.vlllage.com.closer.store.StringListJsonConverter
import io.objectbox.annotation.Convert
import io.objectbox.annotation.Entity

@Entity
class Phone : BaseObject() {
    var name: String? = null
    var status: String? = null
    var introduction: String? = null
    var offtime: String? = null
    var history: String? = null
    var photo: String? = null
    var latitude: Double? = null
    var longitude: Double? = null
    var verified: Boolean? = null

    @Convert(converter = StringListJsonConverter::class, dbType = String::class)
    var goals: List<String>? = null

    @Convert(converter = StringListJsonConverter::class, dbType = String::class)
    var lifestyles: List<String>? = null
}
